.gitignore

```diff
@@ -22,3 +22,9 @@
 
 # Mac Stuff.
 .DS_Store
+
+# Win Stuff
+configure.pyc
```

Comment:
Where is this file generated? If it's generated in root folder (mediasoup folder) please indicate it with `/configure.pyc` instead.

Reply:
generated by python. the path is `/worker/scripts/configure.pyc`. already add explicit path.